### ISSUE TITLE
Support custom list of txt files for the names of s3 buckets

### ISSUE THREAD

**Issue by ajutamangdev** [ajutamangdev/S3Khoj#5]
User Behaviour and expectations
```
S3Khoj -l buckets.txt  
```

**Comment by xcypher78** [comment]
> User Behaviour and expectations
> 
> ```
> S3Khoj -l buckets.txt  
> ```

you will implement ?

**Comment by ajutamangdev** [comment]
Yeah I will. If you are interested to implement, I can consider to assigned with you and review the PR.

**Comment by xcypher78** [comment]
"If you are interested to implement, I can consider to assigned with you and review the PR." sorry i dont understand what you mean

**Comment by xcypher78** [comment]
...

**Comment by ajutamangdev** [comment]
Hey, i mean I am planning to implement this one feature. But if you wanna contribute, you can also make pr for this and i will considered to review and help on it.

**Comment by xcypher78** [comment]
no thanks
